src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,8 +1,49 @@
 package edu.stanford.dlss.was;
 
+import java.io.*;
+import java.util.Properties;
+
 public class WasapiDownloader
 {
-    public static void main(String[] args) {
-        System.out.println("I am WasapiDownloader. Hear me roar. Woo.");
+  public static final String SETTINGS_FILE_LOCATION = "config/settings.properties";
+  public Properties settings = null;
+
+  public WasapiDownloader() {
+    loadPropertiesFile();
+  }
+
+  private void loadPropertiesFile() {
+    if (settings == null)
```

Comment:
It seems unlikely we would want to read the settings file more than once ... this is a kind of stupid way of memo-izing the properties we get from the file.   It also means we have to read the file before we parse the command line options.